### ISSUE TITLE
multiline jsx doesn't work

### ISSUE THREAD

**Issue by marcuskwan** [aaron-bond/better-comments#175]
{/*? why is products and addItem wrapped in object*/} 	 
 {/*
	? why is products and addItem wrapped in object
	*/}

both of these don't work,

**Comment by KieranO547** [comment]
This still seems to be broken, as a work around incase it helps (for the fact that linter in react wants these comments and default for vscode, in react return is set to the multiline), you can get it to work by adding `//` at the start of the line, so these work:        
`{/* // ! test */}`
```
{/*
 // ! test
*/}
```

**Comment by marcuskwan** [comment]
> This still seems to be broken, as a work around incase it helps (for the fact that linter in react wants these comments and default for vscode, in react return is set to the multiline), you can get it to work by adding `//` at the start of the line, so these work:
> `{/* // ! test */}`
> 
> ```
> {/*
>  // ! test
> */}
> ```

cool, thanks !

**Comment by KhimairaCrypto** [comment]
Any news regarding fixing this issue? this issue was raised in 2019 more than  2 years ago!

**Comment by kieran-osgood** [comment]
@KhimairaCrypto Considering the number of issues/PR's open, and the fact that the last commit was 14 months ago it's safe to assume that chasing up any kinds of issues is not likely going to go anywhere soon - Open Source takes a lot of time and resources - your best bet would be to fork it and add the support yourself (it has an MIT license so you are free to copy and re-release it if you're able to take on maintaining it!)

**Comment by KhimairaCrypto** [comment]
@kieran-osgood that is a valid point! unfortunately I do not have the capacity to fix every single tool that I currently use :-(. This tools is more in the nice to have category than in the must have one!

**Comment by aaron-bond** [comment]
@kieran-osgood yep, this is pretty bang on. This repo took a lot of time to manage and it got to a "good enough" state where I didn't feel so bad letting it stagnate. I still keep an eye for any outrageous bugs popping up, but I have too many other things to prioritize ATM.

As you've said though, the code is 100% free for anyone to use. Have at it!

**Comment by kieran-osgood** [comment]
@aaron-bond Totally understand - and appreciate the work you put in! Have a great day
